app/views/layouts/hero.html.slim

```diff
@@ -26,7 +26,7 @@ html.govuk-template lang='en'
           = yield :hero
 
       .dfe-width-container
-        #main-content.govuk-main-wrapper
+        .govuk-main-wrapper
```

Comment:
@martikat see https://github.com/alphagov/govuk-frontend/issues/1379, confirm my suspicions and make appropriate changes. @instantrick you should retest any code changes.